src/components/organisms/ChatSidebar/ChatSidebar.scss

```diff
@@ -11,7 +11,7 @@ $header-height: 45px;
   top: 66px;
   right: 0;
   width: 360px;
-  height: calc(100vh - 96px);
+  height: calc(100vh - 66px);
```

Comment:
What does this relate to? What did the `96px` represent, and what does the `66px` now represent? This seems like a place where we should potentially be using variables from `scss/constants`

I think maybe it is what is described in https://github.com/sparkletown/internal-sparkle-issues/issues/233#issuecomment-776502846 ?